### DISCUSSION TITLE
How to export and import settings like the device list (on a new device) android

### DISCUSSION THREAD

**Discussion by jnpf** [emericg/WatchFlower#72] [Q&A] [answered]
I have a friend who is caring my plants while I'm out of home. I try to send him my list of plants so he can use the app the same way I do.

I already know there is no feature for this in the android app right know, but is there a way for geeks? Like copying a specific system file from a to b or something like that.

Thanks and best regards

**Comment by emericg** [comment, marked as answer]
Hi Jan.
No sorry there isn't an advanced mode to save and restore database ^^

On desktop you can indeed just copy/past the database file, that's simply stored in a hidden directory. Unfortunately that isn't possible on Android and iOS.
I'm considering adding a hidden screen in the application with an import/export process, we'll see :p

**Comment by jnpf** [comment]
That would be nice, especially for users with more than 10+ plants (I bet the most of your users).

Thanks a lot
